src/java/picard/vcf/CollectVariantCallingMetrics.java

```diff
@@ -103,21 +103,14 @@ protected int doWork() {
 
         final VariantProcessor.Builder<CallingMetricAccumulator, CallingMetricAccumulator.Result> builder =
                 VariantProcessor.Builder
-                        .generatingAccumulatorsBy(
-                                new VariantProcessor.AccumulatorGenerator<CallingMetricAccumulator, CallingMetricAccumulator.Result>() {
-                                    @Override
-                                    public CallingMetricAccumulator build() {
-                                        return new CallingMetricAccumulator(dbsnp);
-                                    }
-                                })
-                        .combiningResultsBy(
-                                new VariantProcessor.ResultMerger<CallingMetricAccumulator.Result>() {
-                                    @Override
-                                    public CallingMetricAccumulator.Result merge(final Collection<CallingMetricAccumulator.Result>
-                                                                                         resultsToReduce) {
-                                        return CallingMetricAccumulator.Result.merge(resultsToReduce);
-                                    }
-                                })
+                        .generatingAccumulatorsBy(() -> {
+                            if (GVCF_INPUT) {
```

Comment:
Could be reduced to
```java
return GVCF_INPUT ? new GvcfMetricAccumulator(dbsnp) : new CallingMetricAccumulator(dbsnp)
```
which would let you get rid of all three pairs of braces.